scripts/m18_submission_checker/aida_m18_submission_checker.py

```diff
@@ -0,0 +1,381 @@
+        return None
+
+def get_archive_submit_ready_status_values(task_type, archive_member_names):
+    """
+    Prereq: ARCHIVE_FILE_NAME, and ARCHIVE_NAMES have been set
+
+    :return: total number of ttl files found, a dict of ttl counts per valid directory, and a dict of ttl counts per invalid directory
+    :rtype: int, dict {'dirname':int,...}, dict {'dirname':int,...}
+    """
+    ttl_valid_count_dict = {}
+    ttl_invalid_count_dict = {}
+    ttls_total_count = 0
+
+    valid_dir_check = None
+
+    # function pointer for which type of validity checking we want to do
+    if task_type == '1a' or task_type == '2':
+        valid_dir_check = do_TA1a_2_check
+    elif task_type == '1b':
+        valid_dir_check = do_TA1b_check
+    elif task_type == '3':
+        valid_dir_check = do_TA3_check
+
+    for name in archive_member_names:
+
+        # split into levels
+        path_items = re.split('/', name)
+
+        if name.endswith('.ttl'):
+
+            ttls_total_count += 1
+            file_dir_status = valid_dir_check(path_items)
+
+            dir = os.path.dirname(name) + '/'
+
+            if file_dir_status:
+                #get value for the current ttl dir, if it dosen't exist then add it to the dict
+                curr_val = ttl_valid_count_dict.setdefault(dir, 0)
+                curr_val += 1
+                ttl_valid_count_dict[dir] = curr_val
+
+            else:
+                #get value for the current ttl dir, if it dosen't exist then add it to the dict
+                curr_val = ttl_invalid_count_dict.setdefault(dir, 0)
+                curr_val += 1
+                ttl_invalid_count_dict[dir] = curr_val
+
+    return ttls_total_count, ttl_valid_count_dict, ttl_invalid_count_dict
+
+def do_TA1a_2_check(path_items):
+    """
+    Check the path in the archive to ensure the path is valid
+    TASK 1a Directory structure:
+    <TA1aperformer>_<run>
+        NIST
+            <document_id>.ttl                  1a: (1 to X); 2: (1)
+		INTER-TA
+		    <document_id>.ttl                  1a: (0 to X); 2: (0 or 1)
+
+
+    Ignores non ttl files.
+    Prints number of ttl files found, and number of ttl files that are in need to be checked.
+
+    :param path_items: The path to the .ttl file that needs to be checked
+    :return: 'NIST' if .ttl file is in the ../NIST/ directory, 'INTER-TA' if in the ../INTER-TA/ directory or None if not in either directory
+    :rtype: string or None
+    """
+
+    # validate that the ttl file is in an acceptable directory
+    # rule: ttl needs to be in 2nd level -- aka 3rd item in list['name', 'NIST', 'valid.ttl']
+    # rule: ttl needs to be in directory named NIST, or INTER-TA
+
+    try:
+        if ((path_items[-2] == 'NIST' or path_items[-2] == 'INTER-TA') and len(path_items) == 3):
+            return True
+    except:
+        return False
+    return False
+
+
+def do_TA1b_check(path_items):
```

Comment:
I created a submission in the following format

RUN_ID/NIST/Hypothesis_1/{2 .ttl files}
RUN_ID/NIST/Hypothesis_2/{2 .ttl files}
RUN_ID/INTER-TA/{6 .ttl files}

This fails. I would expect it to be a PARTIAL SUCCESS because the NIST folder with nested Hypothesis folders with .ttls is valid. You can get this example in s3 at `aida-validation/m18_checker/Test_1b_1a-Valid.zip1`